backend/app/services/implementations/log_records_service.py

```diff
@@ -98,11 +98,13 @@ def filter_by_flagged(self, flagged):
         print(flagged)
         return f"\nflagged={bool(flagged)}"
 
-    def get_log_records(self, page_number, filters=None):
+    def get_log_records(self, page_number, return_all, filters=None):
         try:
             results_per_page = int(os.getenv("RESULTS_PER_PAGE"))
```

Comment:
can't we move this into the if claise on line 105 as well?

Reply:
I am not sure if I understand, which part do you think should be moved?